lib/Minz/FrontController.php

```diff
@@ -106,7 +106,7 @@ public function run () {
 	*/
 	private function killApp ($txt = '') {
 		if ($txt == '') {
-			$txt = 'See logs files';
+			$txt = 'See log files for the service "journalctl -xeu freshrss" or check in /var/log if the service is not present';
```

Comment:
I see no reason for the discrepancy. ;-)
```suggestion
			$txt = 'See log files for the service "journalctl -xeu freshrss" or check in /var/log if the service is not present.';
```